src/models/__init__.py

```diff
@@ -1,11 +0,0 @@
-from flask_sqlalchemy import SQLAlchemy
-
-# initialize our db
-db = SQLAlchemy()
-
-
-from flask_bcrypt import Bcrypt
-#######
-# existing code remains #
-#######
-bcrypt = Bcrypt()
```

Comment:
What's the second recordexpunge directory for?

Reply:
I removed the redudant dir in `src/backend/` to leave`src/backend/expungeservice/` as introduced in commit 234a6c0and.